src/main/resources/application.yml

```diff
@@ -21,7 +21,7 @@ management:
       base-path: /actuator
       exposure.include: "*"
   server:
-    port: 9000
+    port: ${SERVER_PORT:9000}
```

Comment:
doing the same as in the line 2 in this file.

Reply:
Then it rather should be:   ` port: ${MANAGEMENT_SERVER_PORT:9000}`, but actually both are redundant. They should only be
 
server
    port: 9000
    
management
    server
        port: 9000

Because the env variables SERVER_PORT and MANAGEMENT_SERVER_PORT will override these properties anyway.

Reply:
Thanks for mentioning it. Closing this PR. For future reference: see [this](https://docs.spring.io/spring-boot/docs/1.5.6.RELEASE/reference/html/boot-features-external-config.html#:~:text=you%20can%20use%20underscores).